<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupSettings</class>
 <widget class="QWidget" name="GroupSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>622</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="Wrapper">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>500</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame {
background-color:rgba(16,19,23,255);
border:1px solid white;
border-radius:45px;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1,0">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item alignment="Qt::AlignHCenter|Qt::AlignTop">
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">color:white;
font-size:18px;
border:none;</string>
        </property>
        <property name="text">
         <string>Настройки группы</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QFrame" name="spinner_frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border:none;</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="spinner_layout">
           <item>
            <widget class="QLabel" name="error_label">
             <property name="styleSheet">
              <string notr="true">color:red;
font-size:16px;</string>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="info_label">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">color:white;
font-size:16px;</string>
             </property>
             <property name="text">
              <string>Меняем настройки</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item alignment="Qt::AlignVCenter">
       <widget class="QFrame" name="FriendsList">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">border:none;</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>25</number>
         </property>
         <item>
          <widget class="QFrame" name="change_name">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>7</number>
            </property>
            <property name="leftMargin">
             <number>11</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>11</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="styleSheet">
               <string notr="true">color:white;
font-size:18px;
border:none;</string>
              </property>
              <property name="text">
               <string>Сменить название:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="GroupName">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>250</width>
                <height>0</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QLineEdit{ 
width:250px;
height:30px;
border: 2px solid #323338;
border-radius: 10px;
background-color:#1e1f22;
font-size:16px;
color:#808994;
}

</string>
              </property>
              <property name="inputMask">
               <string/>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="maxLength">
               <number>50</number>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="placeholderText">
               <string>Название группы</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="Wrapper_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>500</width>
             <height>0</height>
            </size>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>20</number>
            </property>
            <property name="bottomMargin">
             <number>10</number>
            </property>
            <item alignment="Qt::AlignVCenter">
             <widget class="QCheckBox" name="is_private">
              <property name="styleSheet">
               <string notr="true">QCheckBox {
color:white;
font-size:20px;
}

QCheckBox::indicator {
    width: 18px;
    height: 18px;
    border-radius: 9px;
    border: 2px solid orange;
    background: transparent;
}

QCheckBox::indicator:hover {
    border: 2px solid #rgb(255, 170, 0);
}

QCheckBox::indicator:checked {
    background: orange;
}</string>
              </property>
              <property name="text">
               <string>Приватная группа (только по приглашению)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="is_inviteGromAdmin">
              <property name="styleSheet">
               <string notr="true">QCheckBox {
color:white;
font-size:20px;
}

QCheckBox::indicator {
    width: 18px;
    height: 18px;
    border-radius: 9px;
    border: 2px solid orange;
    background: transparent;
}

QCheckBox::indicator:hover {
    border: 2px solid #rgb(255, 170, 0);
}

QCheckBox::indicator:checked {
    background: orange;
}</string>
              </property>
              <property name="text">
               <string>Приглашения только от администратора</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="is_password">
              <property name="styleSheet">
               <string notr="true">QCheckBox {
color:white;
font-size:20px;
}

QCheckBox::indicator {
    width: 18px;
    height: 18px;
    border-radius: 9px;
    border: 2px solid orange;
    background: transparent;
}

QCheckBox::indicator:hover {
    border: 2px solid #rgb(255, 170, 0);
}

QCheckBox::indicator:checked {
    background: orange;
}</string>
              </property>
              <property name="text">
               <string>Вход по паролю</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="change_password_button">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>35</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true"> QPushButton {
border:1px solid orange;
border-radius:15%;
background-color:orange;
color:white;
font-size:18px;
}

QPushButton:hover {
	background-color: rgba(255, 211, 121, 200);
	border:1px solid rgba(255, 211, 121, 200);
}</string>
              </property>
              <property name="text">
               <string>Сменить пароль</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="password_wrapper">
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="leftMargin">
             <number>11</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>11</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="styleSheet">
               <string notr="true">color:white;
font-size:18px;</string>
              </property>
              <property name="text">
               <string>Введите новый пароль:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="Password_input">
              <property name="minimumSize">
               <size>
                <width>250</width>
                <height>0</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QLineEdit{ 
width:250px;
height:30px;
border: 2px solid #323338;
border-radius: 10px;
background-color:#1e1f22;
font-size:16px;
color:#808994;
text-align:left;
padding-left:27px;
}

</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="placeholderText">
               <string>Пароль</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="styleSheet">
               <string notr="true">color:white;
font-size:18px;</string>
              </property>
              <property name="text">
               <string>Повторить пароль:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="RepeatPassword_input">
              <property name="minimumSize">
               <size>
                <width>250</width>
                <height>0</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QLineEdit{ 
width:250px;
height:30px;
border: 2px solid #323338;
border-radius: 10px;
background-color:#1e1f22;
font-size:16px;
color:#808994;
text-align:left;
padding-left:27px;
}

</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="placeholderText">
               <string>Повторить пароль</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="save_changes_button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>151</width>
          <height>50</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true"> QPushButton {
border:1px solid orange;
border-radius:25%;
background-color:orange;
color:white;
font-size:18px;
}

QPushButton:hover {
	background-color: rgba(255, 211, 121, 200);
	border:1px solid rgba(255, 211, 121, 200);
}</string>
        </property>
        <property name="text">
         <string>Сохранить</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
